src/config/config.ts

```diff
@@ -193,6 +201,12 @@ const config = convict({
         format: "required-string",
         default: "",
       },
+      ffDeprecateSiteQueues: {
+        doc: "Whether the queues are deprecated",
```

Comment:
docs should aim to convey contextual/implicit info that isn't obvious from the name. side-note, i think the `ff` prefix is for the raw env var? here, we could group it under a separate tag (ie, `featureFlag`) if that makes more sense

```suggestion
      shouldDeprecateSiteQueues: {
        doc: "Whether queues should be used instead of dynamoDb",
```

Reply:
0694a084